tests/integration/multi_table/test_hma.py

```diff
@@ -604,6 +604,21 @@ def test_use_own_data_using_hma(tmp_path):
         sdtype='id',
         regex_format='HID_[0-9]{3,4}'
     )
+    metadata.update_column(
+        table_name='hotels',
+        column_name='city',
+        sdtype='categorical',
+    )
+    metadata.update_column(
+        table_name='hotels',
+        column_name='state',
+        sdtype='categorical',
+    )
+    metadata.update_column(
+        table_name='hotels',
+        column_name='classification',
+        sdtype='categorical',
+    )
```

Comment:
Are we updating this because now they are no longer `categories` but `ids` ?

Reply:
They were detected as `unknown` but it was mostly because the demo dataset here is small and so the condition `Elif the cardinality is less than round(len / 5), sdtype is categorical` was not reached. I discussed it with Neha and we decided that it was fine like this.